### PR TITLE
chore: change changesets config

### PR DESCRIPTION
**Pull request by ota-meshi**
See https://github.com/ota-meshi/eslint-plugin-svelte/pull/235#issuecomment-1223395556

### CODE REVIEW

**Review by JounQin — APPROVED** [review]
Yeah, this is exactly what I mean!

File: package.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
     "ts": "node -r esbuild-register",
     "typecov": "type-coverage",
     "update": "yarn ts ./tools/update.ts && yarn format-for-gen-file",
-    "version": "env-cmd -e version yarn update && git add ."
+    "version": "env-cmd -e version yarn update && changeset version"
```
Comment (JounQin) [inline package.json:54]: Like `prerelease`,  we can extract `preversion`,  but it's not required, of course.

Comment (ota-meshi) [reply package.json:54]: We can't use preversion because the script needs to be run after package.json is updated.

Comment (JounQin) [reply package.json:54]: Then or `postversion`?

**Pull request merged**